bpf/lib/policy.h

```diff
@@ -194,11 +198,11 @@ static inline int policy_can_egress6(struct __sk_buff *skb,
 	struct remote_endpoint_info *info;
 	__u16 identity = 0;
 
-	info = lookup_ip6_remote_endpoint(&tuple->daddr);
+	info = lookup_ip6_remote_endpoint(&tuple->saddr);
```

Comment:
As a point of interest, I think this just works because we swap the src/dst for connection tracking. We do actually want the daddr, so maybe we could make this clearer by factoring it into a `__be32 *daddr` variable on the stack with a short comment describing it, eg:

```
/* For outgoing connection, connection tracking swaps the src/dst. We want the packet dst. */
__be32 *daddr = &tuple->saddr;
```